client/src/components/Tool/ToolForm.vue

```diff
@@ -3,6 +3,9 @@
         <b-alert :show="messageShow" :variant="messageVariant">
             {{ messageText }}
         </b-alert>
+        <b-alert v-if="!showLoading && !canMutateHistory" show variant="warning">
```

Comment:
We probably need the same in the workflow run form, the upload dialog and the collection builders ?

Reply:
I've added similar checks to the workflow run and the upload dialog. Is there a way to reach the collection builders without using the upload dialog?
I hope all cases are covered now, in the worst case it will fail after trying but there is a lot of feedback now on what kind of history we are dealing with.